tests/plugin/test_override_by_external_plugins.py

```diff
@@ -1,8 +1,24 @@
+from tempfile import NamedTemporaryFile
 from textwrap import dedent
 
 import pytest
 
 
+@pytest.mark.parametrize("content", ["[corrupted", "[corrupted"])
+def test_corrupted_config_raises_human_friendly_error(
```

Comment:
This seems to be a duplicate of `test_config.py::test_corrupted_config_raises_human_friendly_error`

Reply:
Good catch, it's from debugging times 🙈